tests/workflows/test_main_snapshot.py

```
@@ -0,0 +1,185 @@
+            f"expected 1 argument to _collect_manifest_lines, got {len(call_args)}"
+        )
+        manifest_path = call_args[0]
+        assert str(manifest_path) == "installer/dependency-binaries.toml", (
+            f"expected default manifest path, got {manifest_path}"
+        )
+
+
+class TestSnapshotOutput:
+    """Snapshot-style tests for TSV output against the real manifest."""
+
+    def test_real_manifest_output_structure(self, tmp_path: Path) -> None:
```

Ruff (0.15.9)

[warning] Unused method argument: `tmp_path`

(ARG002)